include/gz/math/Inertial.hh

```diff
@@ -328,6 +413,9 @@ namespace gz
       /// \brief Pose offset of center of mass reference frame relative
       /// to a base frame.
       private: Pose3<T> pose;
+
+      /// \brief Fluid added mass.
+      private: Matrix6<T> addedMass;
```

Comment:
can this be a `std::optional<Matrix6<T>`?  it will be easier to detect if it has never been set

Reply:
Good idea, done in [b22d60f](https://github.com/gazebosim/gz-math/pull/459/commits/b22d60f3106734fad452fd5fab6975edb34df8a7)